src/sonic-frr-mgmt-framework/frrcfgd/frrcfgd.py

```diff
@@ -2258,7 +2269,7 @@ def __init__(self):
         syslog.syslog(syslog.LOG_DEBUG, 'Init Cached DB data')
         for key, entry in self.table_data_cache.items():
             syslog.syslog(syslog.LOG_DEBUG, '  %-20s : %s' % (key, entry))
-        if self.config_mode == "unified":
+        if self.config_mode == "unified" or self.config_mode == "separated":
```

Comment:
As I understand it, 

1. The initial configuration generated via j2 files to individual proto conf files will populate frr initially.
2. As the configuration is loaded in the DB, frrcfgd will process each event and again populate frr (same configuration)
3. This change with addition of the check `"self.config_mode == "separated"` will make another attempt to configure frr

Step 1 & 2 is the current implementation
Step 1 - will make the configuration available to frr much faster than the other 2.
Step 3 - is redundant.

Reply:
That means the same config handling would be coded in j2 and frrcfgd.py. That requires changing in two different places. Do we really want to take this approach?

Reply:
are we ok with slow config load and 3 times config updates?

Reply:
I am open for suggestions. Here are the issues I have hit

1. When using "unified" mode, "fpm address 127.0.0.1" is not applied. Does anyone know how this should be applied?
2. When using "separated" mode, new added configurations would be not applied at init time. It seems the initial j2 files are used. My concern is if that means all configurations would be created two copies of handling, one in frrcfgd.py for run time changing, and one in j2. How do we guarantee the consistency in developing. 


If someone could help to figure out how "fpm address 127.0.0.1" get applied in "unified" mode, I would move back to that mode which seems cleaner.

Reply:
moved to unified mode as suggested by Venkat.